macaw-aarch32/src/Data/Macaw/ARM/ARMReg.hs

```diff
@@ -107,9 +124,67 @@ locToRegTH (SA.Location globalRef) = do
       [| ARMDummyReg |]
     _tp -> [| error $ "locToRegTH undefined for unrecognized location: " <> $(return $ TH.LitE (TH.StringL refName)) |]
 
+r0 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r0 = ARMGlobalBV (ASL.knownGlobalRef @"_R0")
+
+r1 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r1 = ARMGlobalBV (ASL.knownGlobalRef @"_R1")
+
+r2 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r2 = ARMGlobalBV (ASL.knownGlobalRef @"_R2")
+
+r3 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r3 = ARMGlobalBV (ASL.knownGlobalRef @"_R3")
+
+r4 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r4 = ARMGlobalBV (ASL.knownGlobalRef @"_R4")
+
+r5 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r5 = ARMGlobalBV (ASL.knownGlobalRef @"_R5")
+
+r6 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r6 = ARMGlobalBV (ASL.knownGlobalRef @"_R6")
+
+r7 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r7 = ARMGlobalBV (ASL.knownGlobalRef @"_R7")
+
+r8 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r8 = ARMGlobalBV (ASL.knownGlobalRef @"_R8")
+
+r9 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r9 = ARMGlobalBV (ASL.knownGlobalRef @"_R9")
+
+r10 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r10 = ARMGlobalBV (ASL.knownGlobalRef @"_R10")
+
+r11 :: (w ~ MC.RegAddrWidth ARMReg, 1 <= w) => ARMReg (MT.BVType w)
+r11 = ARMGlobalBV (ASL.knownGlobalRef @"_R11")
```

Comment:
In pursuit of API parity with [`Data.Macaw.AArch32.Symbolic.Regs`](https://github.com/GaloisInc/macaw/blob/a1e36fde2b191128e7361f98df45927fb867bd51/macaw-aarch32-symbolic/src/Data/Macaw/AArch32/Symbolic/Regs.hs), it would be worth adding an [`fp`](https://github.com/GaloisInc/macaw/blob/a1e36fde2b191128e7361f98df45927fb867bd51/macaw-aarch32-symbolic/src/Data/Macaw/AArch32/Symbolic/Regs.hs#L211-L213) alias for this.